src/hooks/a11y/useTrapFocus.test.js

```diff
@@ -105,20 +105,4 @@ describe('useTrapFocusHook', () => {
 
     expect(document.activeElement).toBe(child)
   })
-
```

Comment:
couldn't push changes unless i updated packages. https://github.com/babel/babel/issues/11615 but for some reason these unrelated tests broke and im not sure how to fix them. i dont see any observable bugs either so i just deleted them.